README.md

```diff
@@ -163,82 +142,13 @@ Then the environment variable `SCHEMATIC_KEY_VALUE` needs to be set. The value o
 
 ### Hooks
 
-This tool has three hooks that extending code can plug in to. An example of a project using some of these hooks is the [Schematic plugin for AmNav](https://github.com/nerds-and-company/schematic-amnav).
-
-
-#### registerMigrationService
-
-<table>
-<tr><td>Called by</td><td><code>NerdsAndCompany\Schematic\Services\Schematic::importFromYaml()</code>,  <code>NerdsAndCompany\Schematic\Services\Schematic::exportToYaml()</code></td></tr>
-<tr><td>Return</td><td>An array where the keys are a name and the values are a Migration Service.</td></tr>
-</table>
-
-Gives plugins a chance to register their own Migration Services to Schematic in order to import or exports their own data.
+There are no hooks available at this time.
```

Comment:
Will we add them as we need them?

Reply:
yes, I've currently hardcoded mapper and datatypes config in the Schematic class.
Converters are only search for in our namespace atm.
We could probably make this configurable in the future.